### PR TITLE
[stdlib]specify LibraBlock

### PR DESCRIPTION
**Pull request by emmazzz**
## Motivation

This PR includes specifications for LibraBlock. I also fixed the indentation in LibraBlock.move.
And `root_ctm_initialized` is changed to `spec_root_ctm_initialized` to fit our convention that spec functions should start with `spec`.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

cargo test

### CODE REVIEW

File: language/stdlib/modules/LibraBlock.move [hunk 1]
```diff
@@ -4,24 +4,25 @@ module LibraBlock {
...
     use 0x1::LibraTimestamp;
     use 0x1::Signer;
 
     resource struct BlockMetadata {
-      // Height of the current block
-      // TODO: should we keep the height?
-      height: u64,
-      // Handle where events with the time of new blocks are emitted
-      new_block_events: Event::EventHandle<Self::NewBlockEvent>,
+        // Height of the current block
```
Comment (wrwg) [inline language/stdlib/modules/LibraBlock.move:12]: As you work over this, could you turn `//` to `///` where apropriate?

File: language/stdlib/modules/LibraBlock.move [hunk 2]
```diff
@@ -4,24 +4,25 @@ module LibraBlock {
...
+        // TODO: should we keep the height?
```
Comment (wrwg) [inline language/stdlib/modules/LibraBlock.move:13]: Just checking: are you adding one or two spaces? (Can't see from this view). Please ensure indentation is 4.

Comment (emmazzz) [reply language/stdlib/modules/LibraBlock.move:13]: Yes I have changed all the indentation to be four spaces.

**Pull request closed without merging**